frontend/src/scenes/data-management/ingestion-warnings/IngestionWarningsView.tsx

```diff
@@ -17,14 +17,16 @@ export const scene: SceneExport = {
 
 const WARNING_TYPE_TO_DESCRIPTION = {
     cannot_merge_already_identified: 'Refused to merge an already identified user via $identify or $create_alias call',
+    cannot_merge_with_illegal_distinct_id:
```

Comment:
For both this and `cannot_merge_already_identified` could we also link to relevant documentation you're about to add as to how to fix the issue?

As-is users would not know what to do about thos.